pylib/mozhginfo/mozhginfo/push.py

```diff
@@ -0,0 +1,62 @@
+class Push(object):
+    """This represents a Push"""
+
+    def __init__(self, date):
+        """
+            We set Push object default as dict, and we will
+            give each push a push id when we init it.
+        """
+        self._id, self._push = date.popitem()
+
+    @property
+    def id(self):
+        """
+            Property for getting the Push ID of a push.
+            >>> push.id
+            123456
+        """
+        return self._id
+
+    @property
+    def changesets(self):
+        """
+            Property for getting the changesets list of a push.
+            >>> push.changesets
+            >>> ["123456", "sad123"]
+        """
+        return self._push.get('changesets', None)
+
+    @property
+    def first_changeset(self):
```

Comment:
Don't add more properties than we need.
Let's find reasons for needing this.

Using something like this should be evident:
p = Push(12345)
p.changesets[0] # first
p.changesets[-1] # last